doc/rst/source/pstext.rst

```diff
@@ -79,12 +73,15 @@ To add a typeset figure caption for a 3-inch wide illustration, use
     of cities where it is @\_impossible@\_ to get any good Thai food; these are to be avoided.
     EOF
 
-To add a text without using input data but only the fixed text option
-
-   ::
+To add a text without using input data but only the fixed text option::
 
     gmt pstext -R0/10/0/10 -JX14 -Baf -F+f40+cTC+t"Inner Title" -D0/-0.5 -P > figure.ps
 
+
+To place a line containing a Latex equation, try::
+
+    echo 3 3 'Use @$\Delta g = 2\pi\rho Gh@$' | gmt pstext -R0/6/0/6 -JX15c -B -F+f32p+a30 -P > map.ps
```

Comment:
In classic mode, we need **-Baf**, otherwise GMT reports `pstext [ERROR]: Found no history for option -`
```suggestion
    echo 3 3 'Use @$\Delta g = 2\pi\rho Gh@$' | gmt pstext -R0/6/0/6 -JX15c -Baf -F+f32p+a30 -P > map.ps
```